lxd/network/driver_ovn.go

```diff
@@ -4365,7 +4365,8 @@ func (n *ovn) PeerCreate(peer api.NetworkPeersPost) error {
 	// Check if there is an existing peer using the same name, or whether there is already a peering (in any
 	// state) to the target network.
 	peers, err := n.state.DB.Cluster.GetNetworkPeers(n.ID())
-	if err == nil {
+	if err != nil {
+		return api.StatusErrorf(http.StatusServiceUnavailable, "Could not validate against existing peers")
```

Comment:
Oh whoops thats one of mine :(

I think we should use http.StatusInternalServerError here (or just return the error, which will also default to internal server error status code) as we use that everywhere else for DB errors.